plugins/gaming.py

```diff
@@ -15,39 +16,53 @@
 
 from cloudbot import hook
 
+# String constants for the coin flip function
+INVALID_NUMBER = "Invalid input {!r}: not a number"
+NO_COIN = "makes a coin flipping motion"
+SINGLE_COIN = "flips a coin and gets {}."
+MANY_COINS = "flips {} coins and gets {} heads and {} tails."
+
+# Pregenerated mean and variance for fudge dice
+FUDGE_MEAN = 0
+FUDGE_VAR = 0.6667
+
+INVALID_ROLL = "Invalid dice roll {!r}"
+
+ROLL_LIMIT = 100  # The maximum number of times to roll or flip before approximating results
+
 whitespace_re = re.compile(r'\s+')
 valid_diceroll = re.compile(r'^([+-]?(?:\d+|\d*d(?:\d+|F))(?:[+-](?:\d+|\d*d(?:\d+|F)))*)( .+)?$', re.I)
 sign_re = re.compile(r'[+-]?(?:\d*d)?(?:\d+|F)', re.I)
 split_re = re.compile(r'([\d+-]*)d?(F|\d*)', re.I)
 
 
-def clamp(n, min_value, max_value):
-    """Restricts a number to a certain range of values,
-    returning the min or max value if the value is too small or large, respectively
-    :param n: The value to clamp
-    :param min_value: The minimum possible value
-    :param max_value: The maximum possible value
-    :return: The clamped value
-    """
-    return min(max(n, min_value), max_value)
-
-
-def n_rolls(count, n):
+def n_rolls(roll_cnt, sides):
     """roll an n-sided die count times
-    :type count: int
-    :type n: int | str
+
+    :type roll_cnt: int
+    :type sides: int | str
```

Comment:
```suggestion
    :type sides: int | str
    :rtype: list[str]
```

Reply:
The return type of nrolls is a list of ints, not strings.

Reply:
Yes I realized just now that I mistyped that